extensions/amp-bind/amp-bind.md

```diff
@@ -991,7 +991,7 @@ Change the `width` and `height` using the `[width]` and `[height]` attributes.
 
 **Accessibility states and properties**
 
-Use to dynamically update information available to assistive technologies, such as screen readers. All `[aria-*]` and `[data-*]` are bindable. See the [full list here](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties).
+Use to dynamically update information available to assistive technologies, such as screen readers. All `[aria-*]` and `[data-*]` attributes (except `data-vars-*`) are bindable. See the [full list here](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties).
```

Comment:
Is this true? As far as I can tell, very few `data-*` attributes are bindable. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/0.1/bind-validator.js.

It may be more correct to delete the `[data-*]` part.

Reply:
Good point! I changed the documentation to reference the next paragraph.